web/themes/custom/recipe_theme/templates/views/views-view-unformatted--recipe-list--block-2.html.twig

```diff
@@ -0,0 +1,33 @@
+{#
```

Comment:
Let's give this block (`block-2`) and all displays inside the `recipe_list` view a more meaningful machine name. You can change the machine name of a view display at the advanced section when editing a view.

If this block is used somewhere (e.g. on the landing page's view display or the block layout) after changing the machine name it will have to be re-added and re-exported.

Reply:
Done! Will push updates in a few minutes.